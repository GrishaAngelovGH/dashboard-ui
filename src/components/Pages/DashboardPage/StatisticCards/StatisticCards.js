import StatisticCard from './StatisticCard'
import { Row } from 'antd'

const StatisticCards = () => {
    const statisticData = [
        { id: 1, title: 'Active Products', value: 11.28, increased: true },
        { id: 2, title: 'Average Downtime', value: 9.30, increased: false },
        { id: 3, title: 'Income', value: 25.48, increased: true },
        { id: 4, title: 'Idling', value: 10.36, increased: false },
        { id: 5, title: 'Active Users', value: 17.74, increased: true }
    ]

    return (
        <div className='row no-gutters justify-content-center text-center p-3'>
            <div className='col-md-10 ml-auto'>
                <Row gutter={5}>
                    {
                        statisticData.map(v => (
                            <StatisticCard
                                key={v.id}
                                title={v.title}
                                value={v.value}
                                increased={v.increased}
                            />
                        ))
                    }
                </Row>
            </div>
        </div>
    )
}

export default StatisticCards