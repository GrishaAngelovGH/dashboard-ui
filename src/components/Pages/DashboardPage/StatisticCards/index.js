import StatisticCards from './StatisticCards'

export default StatisticCards