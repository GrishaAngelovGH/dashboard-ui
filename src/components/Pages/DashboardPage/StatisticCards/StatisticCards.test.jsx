import { render } from '@testing-library/react'

import StatisticCards from './StatisticCards'

window.matchMedia = window.matchMedia || function () {
    return {
        matches: false,
        addListener: function () { },
        removeListener: function () { }
    }
}

test('should render StatisticCards component', () => {
    const view = render(<StatisticCards />)

    expect(view).toMatchSnapshot()
})