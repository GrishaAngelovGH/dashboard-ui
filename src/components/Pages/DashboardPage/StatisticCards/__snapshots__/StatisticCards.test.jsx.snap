// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render StatisticCards component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row g-0 justify-content-center text-center p-3"
      >
        <div
          class="col-md-10 ml-auto"
        >
          <div
            class="ant-row ant-row-center css-dev-only-do-not-override-19lec04"
            style="margin-left: -2.5px; margin-right: -2.5px;"
          >
            <div
              class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4 css-dev-only-do-not-override-19lec04"
              style="padding-left: 2.5px; padding-right: 2.5px;"
            >
              <div
                class="ant-card ant-card-bordered css-dev-only-do-not-override-19lec04"
              >
                <div
                  class="ant-card-body"
                >
                  <div
                    class="ant-statistic css-dev-only-do-not-override-19lec04"
                  >
                    <div
                      class="ant-statistic-title"
                    >
                      Active Products
                    </div>
                    <div
                      class="ant-statistic-content"
                      style="color: rgb(63, 134, 0);"
                    >
                      <span
                        class="ant-statistic-content-prefix"
                      >
                        <span
                          aria-label="arrow-up"
                          class="anticon anticon-arrow-up"
                          role="img"
                        >
                          <svg
                            aria-hidden="true"
                            data-icon="arrow-up"
                            fill="currentColor"
                            focusable="false"
                            height="1em"
                            viewBox="64 64 896 896"
                            width="1em"
                          >
                            <path
                              d="M868 545.5L536.1 163a31.96 31.96 0 00-48.3 0L156 545.5a7.97 7.97 0 006 13.2h81c4.6 0 9-2 12.1-5.5L474 300.9V864c0 4.4 3.6 8 8 8h60c4.4 0 8-3.6 8-8V300.9l218.9 252.3c3 3.5 7.4 5.5 12.1 5.5h81c6.8 0 10.5-8 6-13.2z"
                            />
                          </svg>
                        </span>
                      </span>
                      <span
                        class="ant-statistic-content-value"
                      >
                        <span
                          class="ant-statistic-content-value-int"
                        >
                          11
                        </span>
                        <span
                          class="ant-statistic-content-value-decimal"
                        >
                          .28
                        </span>
                      </span>
                      <span
                        class="ant-statistic-content-suffix"
                      >
                        %
                      </span>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4 css-dev-only-do-not-override-19lec04"
              style="padding-left: 2.5px; padding-right: 2.5px;"
            >
              <div
                class="ant-card ant-card-bordered css-dev-only-do-not-override-19lec04"
              >
                <div
                  class="ant-card-body"
                >
                  <div
                    class="ant-statistic css-dev-only-do-not-override-19lec04"
                  >
                    <div
                      class="ant-statistic-title"
                    >
                      Average Downtime
                    </div>
                    <div
                      class="ant-statistic-content"
                      style="color: rgb(207, 19, 34);"
                    >
                      <span
                        class="ant-statistic-content-prefix"
                      >
                        <span
                          aria-label="arrow-down"
                          class="anticon anticon-arrow-down"
                          role="img"
                        >
                          <svg
                            aria-hidden="true"
                            data-icon="arrow-down"
                            fill="currentColor"
                            focusable="false"
                            height="1em"
                            viewBox="64 64 896 896"
                            width="1em"
                          >
                            <path
                              d="M862 465.3h-81c-4.6 0-9 2-12.1 5.5L550 723.1V160c0-4.4-3.6-8-8-8h-60c-4.4 0-8 3.6-8 8v563.1L255.1 470.8c-3-3.5-7.4-5.5-12.1-5.5h-81c-6.8 0-10.5 8.1-6 13.2L487.9 861a31.96 31.96 0 0048.3 0L868 478.5c4.5-5.2.8-13.2-6-13.2z"
                            />
                          </svg>
                        </span>
                      </span>
                      <span
                        class="ant-statistic-content-value"
                      >
                        <span
                          class="ant-statistic-content-value-int"
                        >
                          9
                        </span>
                        <span
                          class="ant-statistic-content-value-decimal"
                        >
                          .30
                        </span>
                      </span>
                      <span
                        class="ant-statistic-content-suffix"
                      >
                        %
                      </span>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4 css-dev-only-do-not-override-19lec04"
              style="padding-left: 2.5px; padding-right: 2.5px;"
            >
              <div
                class="ant-card ant-card-bordered css-dev-only-do-not-override-19lec04"
              >
                <div
                  class="ant-card-body"
                >
                  <div
                    class="ant-statistic css-dev-only-do-not-override-19lec04"
                  >
                    <div
                      class="ant-statistic-title"
                    >
                      Income
                    </div>
                    <div
                      class="ant-statistic-content"
                      style="color: rgb(63, 134, 0);"
                    >
                      <span
                        class="ant-statistic-content-prefix"
                      >
                        <span
                          aria-label="arrow-up"
                          class="anticon anticon-arrow-up"
                          role="img"
                        >
                          <svg
                            aria-hidden="true"
                            data-icon="arrow-up"
                            fill="currentColor"
                            focusable="false"
                            height="1em"
                            viewBox="64 64 896 896"
                            width="1em"
                          >
                            <path
                              d="M868 545.5L536.1 163a31.96 31.96 0 00-48.3 0L156 545.5a7.97 7.97 0 006 13.2h81c4.6 0 9-2 12.1-5.5L474 300.9V864c0 4.4 3.6 8 8 8h60c4.4 0 8-3.6 8-8V300.9l218.9 252.3c3 3.5 7.4 5.5 12.1 5.5h81c6.8 0 10.5-8 6-13.2z"
                            />
                          </svg>
                        </span>
                      </span>
                      <span
                        class="ant-statistic-content-value"
                      >
                        <span
                          class="ant-statistic-content-value-int"
                        >
                          25
                        </span>
                        <span
                          class="ant-statistic-content-value-decimal"
                        >
                          .48
                        </span>
                      </span>
                      <span
                        class="ant-statistic-content-suffix"
                      >
                        %
                      </span>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4 css-dev-only-do-not-override-19lec04"
              style="padding-left: 2.5px; padding-right: 2.5px;"
            >
              <div
                class="ant-card ant-card-bordered css-dev-only-do-not-override-19lec04"
              >
                <div
                  class="ant-card-body"
                >
                  <div
                    class="ant-statistic css-dev-only-do-not-override-19lec04"
                  >
                    <div
                      class="ant-statistic-title"
                    >
                      Idling
                    </div>
                    <div
                      class="ant-statistic-content"
                      style="color: rgb(207, 19, 34);"
                    >
                      <span
                        class="ant-statistic-content-prefix"
                      >
                        <span
                          aria-label="arrow-down"
                          class="anticon anticon-arrow-down"
                          role="img"
                        >
                          <svg
                            aria-hidden="true"
                            data-icon="arrow-down"
                            fill="currentColor"
                            focusable="false"
                            height="1em"
                            viewBox="64 64 896 896"
                            width="1em"
                          >
                            <path
                              d="M862 465.3h-81c-4.6 0-9 2-12.1 5.5L550 723.1V160c0-4.4-3.6-8-8-8h-60c-4.4 0-8 3.6-8 8v563.1L255.1 470.8c-3-3.5-7.4-5.5-12.1-5.5h-81c-6.8 0-10.5 8.1-6 13.2L487.9 861a31.96 31.96 0 0048.3 0L868 478.5c4.5-5.2.8-13.2-6-13.2z"
                            />
                          </svg>
                        </span>
                      </span>
                      <span
                        class="ant-statistic-content-value"
                      >
                        <span
                          class="ant-statistic-content-value-int"
                        >
                          10
                        </span>
                        <span
                          class="ant-statistic-content-value-decimal"
                        >
                          .36
                        </span>
                      </span>
                      <span
                        class="ant-statistic-content-suffix"
                      >
                        %
                      </span>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4 css-dev-only-do-not-override-19lec04"
              style="padding-left: 2.5px; padding-right: 2.5px;"
            >
              <div
                class="ant-card ant-card-bordered css-dev-only-do-not-override-19lec04"
              >
                <div
                  class="ant-card-body"
                >
                  <div
                    class="ant-statistic css-dev-only-do-not-override-19lec04"
                  >
                    <div
                      class="ant-statistic-title"
                    >
                      Active Users
                    </div>
                    <div
                      class="ant-statistic-content"
                      style="color: rgb(63, 134, 0);"
                    >
                      <span
                        class="ant-statistic-content-prefix"
                      >
                        <span
                          aria-label="arrow-up"
                          class="anticon anticon-arrow-up"
                          role="img"
                        >
                          <svg
                            aria-hidden="true"
                            data-icon="arrow-up"
                            fill="currentColor"
                            focusable="false"
                            height="1em"
                            viewBox="64 64 896 896"
                            width="1em"
                          >
                            <path
                              d="M868 545.5L536.1 163a31.96 31.96 0 00-48.3 0L156 545.5a7.97 7.97 0 006 13.2h81c4.6 0 9-2 12.1-5.5L474 300.9V864c0 4.4 3.6 8 8 8h60c4.4 0 8-3.6 8-8V300.9l218.9 252.3c3 3.5 7.4 5.5 12.1 5.5h81c6.8 0 10.5-8 6-13.2z"
                            />
                          </svg>
                        </span>
                      </span>
                      <span
                        class="ant-statistic-content-value"
                      >
                        <span
                          class="ant-statistic-content-value-int"
                        >
                          17
                        </span>
                        <span
                          class="ant-statistic-content-value-decimal"
                        >
                          .74
                        </span>
                      </span>
                      <span
                        class="ant-statistic-content-suffix"
                      >
                        %
                      </span>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row g-0 justify-content-center text-center p-3"
    >
      <div
        class="col-md-10 ml-auto"
      >
        <div
          class="ant-row ant-row-center css-dev-only-do-not-override-19lec04"
          style="margin-left: -2.5px; margin-right: -2.5px;"
        >
          <div
            class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4 css-dev-only-do-not-override-19lec04"
            style="padding-left: 2.5px; padding-right: 2.5px;"
          >
            <div
              class="ant-card ant-card-bordered css-dev-only-do-not-override-19lec04"
            >
              <div
                class="ant-card-body"
              >
                <div
                  class="ant-statistic css-dev-only-do-not-override-19lec04"
                >
                  <div
                    class="ant-statistic-title"
                  >
                    Active Products
                  </div>
                  <div
                    class="ant-statistic-content"
                    style="color: rgb(63, 134, 0);"
                  >
                    <span
                      class="ant-statistic-content-prefix"
                    >
                      <span
                        aria-label="arrow-up"
                        class="anticon anticon-arrow-up"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="arrow-up"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M868 545.5L536.1 163a31.96 31.96 0 00-48.3 0L156 545.5a7.97 7.97 0 006 13.2h81c4.6 0 9-2 12.1-5.5L474 300.9V864c0 4.4 3.6 8 8 8h60c4.4 0 8-3.6 8-8V300.9l218.9 252.3c3 3.5 7.4 5.5 12.1 5.5h81c6.8 0 10.5-8 6-13.2z"
                          />
                        </svg>
                      </span>
                    </span>
                    <span
                      class="ant-statistic-content-value"
                    >
                      <span
                        class="ant-statistic-content-value-int"
                      >
                        11
                      </span>
                      <span
                        class="ant-statistic-content-value-decimal"
                      >
                        .28
                      </span>
                    </span>
                    <span
                      class="ant-statistic-content-suffix"
                    >
                      %
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4 css-dev-only-do-not-override-19lec04"
            style="padding-left: 2.5px; padding-right: 2.5px;"
          >
            <div
              class="ant-card ant-card-bordered css-dev-only-do-not-override-19lec04"
            >
              <div
                class="ant-card-body"
              >
                <div
                  class="ant-statistic css-dev-only-do-not-override-19lec04"
                >
                  <div
                    class="ant-statistic-title"
                  >
                    Average Downtime
                  </div>
                  <div
                    class="ant-statistic-content"
                    style="color: rgb(207, 19, 34);"
                  >
                    <span
                      class="ant-statistic-content-prefix"
                    >
                      <span
                        aria-label="arrow-down"
                        class="anticon anticon-arrow-down"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="arrow-down"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M862 465.3h-81c-4.6 0-9 2-12.1 5.5L550 723.1V160c0-4.4-3.6-8-8-8h-60c-4.4 0-8 3.6-8 8v563.1L255.1 470.8c-3-3.5-7.4-5.5-12.1-5.5h-81c-6.8 0-10.5 8.1-6 13.2L487.9 861a31.96 31.96 0 0048.3 0L868 478.5c4.5-5.2.8-13.2-6-13.2z"
                          />
                        </svg>
                      </span>
                    </span>
                    <span
                      class="ant-statistic-content-value"
                    >
                      <span
                        class="ant-statistic-content-value-int"
                      >
                        9
                      </span>
                      <span
                        class="ant-statistic-content-value-decimal"
                      >
                        .30
                      </span>
                    </span>
                    <span
                      class="ant-statistic-content-suffix"
                    >
                      %
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4 css-dev-only-do-not-override-19lec04"
            style="padding-left: 2.5px; padding-right: 2.5px;"
          >
            <div
              class="ant-card ant-card-bordered css-dev-only-do-not-override-19lec04"
            >
              <div
                class="ant-card-body"
              >
                <div
                  class="ant-statistic css-dev-only-do-not-override-19lec04"
                >
                  <div
                    class="ant-statistic-title"
                  >
                    Income
                  </div>
                  <div
                    class="ant-statistic-content"
                    style="color: rgb(63, 134, 0);"
                  >
                    <span
                      class="ant-statistic-content-prefix"
                    >
                      <span
                        aria-label="arrow-up"
                        class="anticon anticon-arrow-up"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="arrow-up"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M868 545.5L536.1 163a31.96 31.96 0 00-48.3 0L156 545.5a7.97 7.97 0 006 13.2h81c4.6 0 9-2 12.1-5.5L474 300.9V864c0 4.4 3.6 8 8 8h60c4.4 0 8-3.6 8-8V300.9l218.9 252.3c3 3.5 7.4 5.5 12.1 5.5h81c6.8 0 10.5-8 6-13.2z"
                          />
                        </svg>
                      </span>
                    </span>
                    <span
                      class="ant-statistic-content-value"
                    >
                      <span
                        class="ant-statistic-content-value-int"
                      >
                        25
                      </span>
                      <span
                        class="ant-statistic-content-value-decimal"
                      >
                        .48
                      </span>
                    </span>
                    <span
                      class="ant-statistic-content-suffix"
                    >
                      %
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4 css-dev-only-do-not-override-19lec04"
            style="padding-left: 2.5px; padding-right: 2.5px;"
          >
            <div
              class="ant-card ant-card-bordered css-dev-only-do-not-override-19lec04"
            >
              <div
                class="ant-card-body"
              >
                <div
                  class="ant-statistic css-dev-only-do-not-override-19lec04"
                >
                  <div
                    class="ant-statistic-title"
                  >
                    Idling
                  </div>
                  <div
                    class="ant-statistic-content"
                    style="color: rgb(207, 19, 34);"
                  >
                    <span
                      class="ant-statistic-content-prefix"
                    >
                      <span
                        aria-label="arrow-down"
                        class="anticon anticon-arrow-down"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="arrow-down"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M862 465.3h-81c-4.6 0-9 2-12.1 5.5L550 723.1V160c0-4.4-3.6-8-8-8h-60c-4.4 0-8 3.6-8 8v563.1L255.1 470.8c-3-3.5-7.4-5.5-12.1-5.5h-81c-6.8 0-10.5 8.1-6 13.2L487.9 861a31.96 31.96 0 0048.3 0L868 478.5c4.5-5.2.8-13.2-6-13.2z"
                          />
                        </svg>
                      </span>
                    </span>
                    <span
                      class="ant-statistic-content-value"
                    >
                      <span
                        class="ant-statistic-content-value-int"
                      >
                        10
                      </span>
                      <span
                        class="ant-statistic-content-value-decimal"
                      >
                        .36
                      </span>
                    </span>
                    <span
                      class="ant-statistic-content-suffix"
                    >
                      %
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4 css-dev-only-do-not-override-19lec04"
            style="padding-left: 2.5px; padding-right: 2.5px;"
          >
            <div
              class="ant-card ant-card-bordered css-dev-only-do-not-override-19lec04"
            >
              <div
                class="ant-card-body"
              >
                <div
                  class="ant-statistic css-dev-only-do-not-override-19lec04"
                >
                  <div
                    class="ant-statistic-title"
                  >
                    Active Users
                  </div>
                  <div
                    class="ant-statistic-content"
                    style="color: rgb(63, 134, 0);"
                  >
                    <span
                      class="ant-statistic-content-prefix"
                    >
                      <span
                        aria-label="arrow-up"
                        class="anticon anticon-arrow-up"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="arrow-up"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M868 545.5L536.1 163a31.96 31.96 0 00-48.3 0L156 545.5a7.97 7.97 0 006 13.2h81c4.6 0 9-2 12.1-5.5L474 300.9V864c0 4.4 3.6 8 8 8h60c4.4 0 8-3.6 8-8V300.9l218.9 252.3c3 3.5 7.4 5.5 12.1 5.5h81c6.8 0 10.5-8 6-13.2z"
                          />
                        </svg>
                      </span>
                    </span>
                    <span
                      class="ant-statistic-content-value"
                    >
                      <span
                        class="ant-statistic-content-value-int"
                      >
                        17
                      </span>
                      <span
                        class="ant-statistic-content-value-decimal"
                      >
                        .74
                      </span>
                    </span>
                    <span
                      class="ant-statistic-content-suffix"
                    >
                      %
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
